docs/source/en/tutorials/inference_with_big_models.md

```diff
@@ -0,0 +1,71 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Working with big models
+
+A modern diffusion model ([SDXL](../using-diffusers/sdxl.md), for example) is not just a single model but a collection of multiple models. SDXL has four different model-level components:
+
+* A variational autoencoder (VAE)
+* Two text encoders
+* A denoiser (which has a UNet architecture)
+
+Usually, the text encoders and the denoiser are bigger and much larger in size compared to the VAE. 
```

Comment:
```suggestion
Usually, the text encoders and UNet are much larger compared to the VAE. 
```

Reply:
Not always a pipeline will have a UNet as a denoiser. Such as PixArt-Alpha. So, will go with:

```suggestion
Usually, the text encoders and the denoiser are much larger compared to the VAE. 
```